packages/appstore/src/components/wallet-cards-carousel/__tests__/wallet-cards-carousel.spec.tsx

```diff
@@ -84,7 +85,25 @@ jest.mock('@deriv/api', () => ({
                 data: {
                     crypto_config: {
                         currencies_config: {
-                            BTC: {},
+                            BTC: {
+                                minimum_withdrawal: 0.00034286,
+                            },
+                            ETH: {
+                                minimum_withdrawal: 0.02728729,
+                            },
+                            LTC: {
+                                minimum_withdrawal: 0.06032091,
+                            },
+                            USD: {},
```

Comment:
Isi neccessary to include `USD` inside when it is empty

Reply:
@aizad-deriv you are right it's not necessary, I'll delete this